packages/eas-cli/src/build/local.ts

```diff
@@ -57,6 +58,7 @@ export async function runLocalBuildAsync(
     const spawnPromise = spawnAsync(command, args, {
       stdio: options.verbose ? 'inherit' : 'pipe',
       env: {
+        ...env,
```

Comment:
I believe that `buildProfile.env` was passed here as `job.builderEnvironment?.env` before, can passing our env vars here alter some behavior of the local build plugin?

https://github.com/expo/eas-build/blob/b6eff7a4bd054c9ff37f631060682bdad7d4679a/packages/local-build-plugin/src/build.ts#L28-L30

Reply:
No, it was never the case: https://github.com/expo/eas-cli/blame/main/packages/eas-cli/src/build/local.ts